js/core/gameSim.js

```diff
@@ -119,6 +119,14 @@ define(["lib/underscore", "util/helpers", "util/random"], function (_, helpers,
     GameSim.prototype.run = function () {
         var out, p, t;
 
+        for(t = 0; t < 2; t++) {
```

Comment:
ba and plusminus values should be initialized in js/core/game.js along with the other stats (player stats on line 536, team stats on line 563)